client/src/pages/SearchResult/SingleSearchResult.tsx

```diff
@@ -71,17 +92,10 @@ export function SingleSearchResultPage() {
               <HeartIcon src={heartFilled} />
```

Comment:
Why are we using `prevState` here, rather than using the `useState` boolean from line 70 directly? You could add a function to toggle `heart`, if `heart` is false and you click on this button, it will `setHeart` to true, and vice versa.

Reply:
Removed. I forget what I was trying to do here.